addons/binding/org.openhab.binding.konnected/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,131 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="konnected"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+	<!-- This is the Konnected Thing-Type -->
+	<thing-type id="module" extensible="switch,actuator,temperature,humidity">
+		<label>The Konnected Alarm Panel</label>
+		<description>The Konnected Module</description>
+		<channels>
+			<channel id="Zone_6" typeId="switch">
+				<label>Zone 6</label>
+				<description>Zone 6 Sensor</description>
+			</channel>
+			<channel id="Zone_7" typeId="actuator">
+				<label>The Out Pin</label>
+			</channel>
+		</channels>
+		<config-description-ref uri="thing-type:konnected:module" />
+	</thing-type>
+	<!-- Zone Channel Type -->
+	<channel-type id="switch">
+		<item-type>Contact</item-type>
+		<label>ON/OFF Switch</label>
+		<description>This zone is a read only switch type zone</description>
+		<state readOnly="true" />
+		<config-description>
+		<parameter name="channel_zone" type="text">
```

Comment:
It would make the textual configuration easier to simply name this parameter "zone" - the shorter, the better.

Reply:
changed